.php_cs.dist

```diff
@@ -0,0 +1,38 @@
+<?php
+
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+       '@PHP56Migration' => true,
+       '@Symfony' => true,
+       '@Symfony:risky' => true,
+       'array_syntax' => ['syntax' => 'short'],
+       'combine_consecutive_unsets' => true,
+       # // one should use PHPUnit methods to set up expected exception instead of annotations
+       'general_phpdoc_annotation_remove' => ['expectedException', 'expectedExceptionMessage', 'expectedExceptionMessageRegExp'],
+       'heredoc_to_nowdoc' => true,
+       'list_syntax' => ['syntax' => 'long'],
+       'method_argument_space' => ['keep_multiple_spaces_after_comma' => false],
+       'no_extra_consecutive_blank_lines' => ['break', 'continue', 'extra', 'return', 'throw', 'use', 'parenthesis_brace_block', 'square_brace_block', 'curly_brace_block'],
+       'no_short_echo_tag' => true,
+       'no_unreachable_default_argument_value' => true,
+       'no_useless_else' => true,
+       'no_useless_return' => true,
+       'ordered_class_elements' => true,
+       'class_attributes_separation' => ['elements' => ['const', 'method', 'property']],
+       'ordered_imports' => true,
+       # 'php_unit_strict' => true,
+       # 'php_unit_test_class_requires_covers' => true,
+       'phpdoc_add_missing_param_annotation' => true,
+       'phpdoc_order' => true,
+       'semicolon_after_instruction' => true,
+       # 'strict_comparison' => true,
+       'strict_param' => true,
```

Comment:
I would suggest to keep these rules sorted by key, makes it easier to find if rules have been added or not.

Reply:
I second @localheinz

Reply:
Fixed